content/tutorials/quickstart/mint-and-burn-nftokens.md

```diff
@@ -20,7 +21,7 @@ This example shows how to:
 3. Delete (Burn) a NFToken.
 
 
-![Test harness with mint NFToken fields](img/quickstart13.png)
+![Test harness with mint NFToken fields](img/quickstart8.png)
```

Comment:
It might be more effective to have descriptive rather than numbered file names for these images so that you don't have to renumber them every time you update the tutorial. Just a suggestion, no action needed on this now.